af04f855-58c4-4db3-a191-45fe33381679/scripts/actions.py

```diff
@@ -732,3 +758,35 @@ def moveOneRandom(group):
 	card.moveTo(me.hand)
 	notify("{} randomly moves {} from their discard to their hand.".format(me, card.name))
 	
+#------------------------------------------------------------------------------
+# Check Deck Tournament Validity
+#------------------------------------------------------------------------------
+
+def CheckTournamentValid(group, x=0, y=0):
+	LegacyLegal=["01-Core Set", "02-Warlord Cycle", "03-The Great Devourer", "04-Planetfall Cycle" "05-Legions of Death", "06-Deathworld Cycle"]
```

Comment:
I'm not sure about making the distinction between champion cards and some 'legacy' format. I understand Apoka doesn't use them and that's fine, is their decision. Is their a current demand for the legacy option that excludes champion cards, or just Apoka is needing a format verifier?

Reply:
Small precision : Legacy option was not asked by Apoka (they only talk about an apoka checker, I took on me to support all formats for currents tournaments that takes places and there are still small events which are only "legacy"). As thoses champions cards were never released, they are not in any current or last tournament which doesn't use any fanmade card. Kinda like the L5R card which come from the books, they are promotional, therefore not legal until released in a cycle.